packages/relay/src/lib/eth.ts

```diff
@@ -1542,6 +1543,30 @@ export class EthImpl implements Eth {
     return logs;
   }
 
+  private async getAccountIdFromAddress(address: string, requestIdPrefix?: string): Promise<string | null> { 
```

Comment:
Doesn't seem with the revert that this is used anymore

Reply:
Removed.